akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/TestInbox.scala

```diff
@@ -6,14 +6,13 @@ package akka.actor.testkit.typed.scaladsl
 
 import akka.actor.{ Address, RootActorPath }
 import akka.actor.typed.ActorRef
-import akka.annotation.{ ApiMayChange, DoNotInherit }
+import akka.annotation.DoNotInherit
 import akka.actor.testkit.typed.internal.TestInboxImpl
 
 import java.util.concurrent.ThreadLocalRandom
 
 import scala.collection.immutable
 
-@ApiMayChange
 object TestInbox {
```

Comment:
Maybe we should keep it for the BehaviorTestKit because I think that needs much love before it's ready, and it could be that it slips the 2.6.0 scope.

We are free to change testkits anyway but it could still be good to signal that it's more unstable.
Btw looks like we don't have the annotation on BehaviorTestKit itself.

Reply:
I'd like to keep this one for now

Reply:
Good idea, i've also added it to the behavior test kit